### DISCUSSION TITLE
Multiple Sliders in the same page

### DISCUSSION THREAD

**Discussion by snpushpi** [jspsych/jsPsych#3111] [General]
Hi all, 
for my project I would like to have multiple sliders on the same page. So I wanna know what modifications I should make in current slider plugin to have that supported. My ideas about what to do are the following - 

1. Inside the parameters do some sort of nested functionality which the survey-likert apparently has
2. Adding a loop in the display_element functionality 
3. Adding a loop to process the results, almost in the similar way of survey-likert. 

Is there anything else I am missing? Besides if anyone has already implemented a custom plugin like that can you please share the code? Would definitely be super helpful :)

**Comment by yukaryote** [comment]
Hey, did you end up figuring this out? Also trying to add multiple sliders

**Reply by snpushpi** [reply]
yes I did, but it was multiple vertical sliders on the same page next to each other under each word, so a little too specific to my experiment design, is that what you want? I can try to find the code if so

**Reply by yukaryote** [reply]
I'm creating multiple sliders that control the XYZ position and orientation of a robot's sensors, and the robot sensor position is updated in real time in a ThreeJS render (I want to collect data on what humans think the optimal placement for robot sensors should be) Upon further thought, I think what I have might be a bit too complex to integrate into jsPsych, but if you have code for creating the sliders that might help anyway!

**Comment by j-fielder** [comment]
Hi @snpushpi, I'd be interested to hear/see how you figured out adding multiple sliders to one page! I'm using the jspsych html slider plug in, and would like to present 2 sliders, 1 on top of the other. Thank you!

**Reply by jodeleeuw** [reply]
hey, there is a survey-slider plugin in [jspsych/jspsych-contrib](https://github.com/jspsych/jspsych-contrib) that can do this now.
